config/environments/production.rb

```diff
@@ -30,7 +30,8 @@
   config.serve_static_files = ENV['RAILS_SERVE_STATIC_FILES'].present?
 
   # Compress JavaScripts and CSS.
-  config.assets.js_compressor = Uglifier.new(harmony: true)
+  config.assets.js_compressor = :terser
+  # config.assets.js_compressor = Uglifier.new(harmony: true)
```

Comment:
I mean I don't really care since this is code is really not going to be expanded further but, can we just remove these old ones or leave a comment?